app/helpers/external_links_helper.rb

```diff
@@ -1,12 +1,16 @@
 module ExternalLinksHelper
-  def edit_url_for(content_id:, publishing_app:, base_path:, document_type:)
+  def edit_url_for(content_id:, publishing_app:, base_path:, document_type:, parent_content_id: '')
```

Comment:
I would expect the default value to be `nil` here in most cases. Is there a benefit to having this default value as an empty string?

Reply:
It's easier to receive an empty string than a nil via the API to then deal with  any `if nil` situations. Setting it to default to the same feels more consistent.